homeassistant/components/devolo_home_network/strings.json

```diff
@@ -66,6 +66,11 @@
       "switch_leds": {
         "name": "Enable LEDs"
       }
+    },
+    "update": {
+      "regular_firmware": {
+        "name": "Firmware Update"
+      }
```

Comment:
Not needed as it can take the device class to make the name "Firmware"

Reply:
Hi @gjohansson-ST , do you have instructions for me, what I need to do? If I just remove the entries, the entity isn't named at all and other integrations I check name it explicitly.

Reply:
You set `translation_key` on the base entity so if you set it to `None` here it will not have a `translation_key` and therefore use the `device_class` as it's name
So on the entity class set
```python
_attr_translation_key = None
```
Then remove that section from `strings.json` as it will take it from the entity component instead.

Reply:
I added a comment where you need to put in in `__init__`

Reply:
Interestingly, that's not doing the trick. The update entity is still named like the device ...

Reply:
I rebased, but still it's not working. So I pushed my changes so that you can have a look. Maybe I made a stupid mistake ...

Reply:
I guess, we are talking about different things. I expected the entitiy to be named "Firmware", instead it is called like the device:

![grafik](https://github.com/home-assistant/core/assets/20171515/d96cbcb0-2b87-4126-baf2-0524fbd07a0c)

I will fix the tests as soon as we understood the issue here.

Reply:
Hi @gjohansson-ST , do you have some further advice for me and maybe also a hint for me, how I can update the device registry with the new version? Currently the old version stays there until Home Assistant is restarted.

Reply:
Hi @gjohansson-ST , #98579 added what we are aiming at here. So I rebased to profit from the changes. Could you have a look again, please?